src/utils/getAssetsUrl.test.ts

```diff
@@ -1,10 +1,36 @@
 import { ASSETS_URL } from './constants';
 import { getAssetsUrl } from './getAssetsUrl';
 
+const uploadCareImage = {
```

Comment:
Uploadcare is written as a single word, let's not mess up their brand name :)
Please update to `uploadcare` or `Uploadcare` across your code.